### ISSUE TITLE
Fantastic Voyage old stew glitch

### ISSUE THREAD

**Issue by Phatzzo** [Wynncraft/Issues#8016]
My quest book tells me to turn in the stew to relend but every time I try he says "If you need to create something, there are utilities on the beach. I know that home is a place we shall once again reach." Even though I've already created the stew. Please help!

**Comment by LordHelmchen98** [comment]
Im going to assume you lost the stew. If so the command `/fixquests` should give you the stew back so you can talk to Relend.
Tell us if it worked or if your problem is different.

**Comment by Phatzzo** [comment]
No, the stew is in my hand and he still wont accept it.

**Comment by Pyromanical** [comment]
Have you tried talking to the cauldron again?

**Comment by Phatzzo** [comment]
Yes, the cooking pot says you have already made the stew check your quest book on what to do next

**Comment by NicBOMB** [comment]
I have the exact same issue. I tried switching server and nothing changed. I have screenshots! It is an album of photos involving the problem.  http://imgur.com/a/8fhvE

**Comment by NicBOMB** [comment]
I'm just sad that I can't use the 21% LB Pendant of Prosperity that I bought on my archer now ;-;
If I could just skip the mission for the quest requirement and not even get any rewards (xp or items included) I would be happy.

**Comment by NicBOMB** [comment]
Bump, please someone respond ;-;
